repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/FieldTypeUtils.java

```diff
@@ -32,6 +32,7 @@
     public static final String PREDEFINED_COMPLEX_CHANGE_ORGANISATION_REQUEST = "ChangeOrganisationRequest";
```

Comment:
Do we need to add a test in FieldTypeUtilsTest for this new field? or does this fields change any logic of isList() ? May be not

Reply:
we have covered this new field in FieldTypeRepositoryTest and ImportServiceImplTest
and this is not going to change any logic of isList()